modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_claim_service_base.rb

```diff
@@ -39,11 +41,42 @@ def get_error_message(error)
           error.original_body
         elsif error.respond_to? :message
           error.message
-        elsif error.is_a?(String)
+        else
           error
         end
       end
 
+      def get_error_key(error_message)
+        return error_message if error_message.is_a? String
+
+        error_message.dig(:messages, 0, :key) || error_message
```

Comment:
As I think about this more, I wonder if it'd be more helpful to fallback to `error_message.class` instead of `error_message`? That being said, I feel like if the `dig` returns `nil`, then it'd be a `String` anywho and caught by the return.